### ISSUE TITLE
[Bug] Default screen resolution issue

### ISSUE THREAD

**Issue by PopkaAkula** [tsunamayo/Starship-EVO#4483]
### Short Description of the issue:

I take screenshots of my builds often and I prefer doing it with a low fov. However, I have come across a weird bug where the default screen resolution is always the lowest possible one, 1024 x 768 @ 60Hz. As a person with a contemporary monitor it gets tedious having to scroll down the list of resolutions every time I want to change the fov.

![image](https://user-images.githubusercontent.com/37738021/149966188-72f83e08-10de-4116-ac9a-4e4dc27a08bd.png)

It always resets to this resolution after you visit the graphics menu.

### Environment
- Starship EVO version: 22w03a

**Comment by pinesh** [comment]
it's not actually changing the resolution to 1024x768 though. It's just the menu defaulting to the start? changing the fov slider doesn't require any alterations to the screen resolution on my end and despite the menu showing 1024, my in-game resolution is clearly not.

**Comment by PopkaAkula** [comment]
> it's not actually changing the resolution to 1024x768 though. It's just the menu defaulting to the start? changing the fov slider doesn't require any alterations to the screen resolution on my end and despite the menu showing 1024, my in-game resolution is clearly not.

When I leave the menu it changes my resolution to 1024x768 every time, even if I don't change anything. 
This problem doesn't occur on my 1920x1080 monitor which leads me to believe that this is only an issue on higher resolution monitors.

**Comment by pinesh** [comment]
I have the higher resolution and haven't had this happen before but I have now however managed to trigger your issue.

So long as you don't change your startup resolution, altering FOV does not cause your resolution to switch to the one listed in on the menu. However. After altering the resolution, even to itself, the behavior you described occurs, this is resolved by restarting and refraining from changing your resolution from the one the game booted with.

https://user-images.githubusercontent.com/14081996/150013756-bbac5141-05c4-41d3-b6c9-0491505e1b8c.mp4

**Comment by tsunamayo** [comment]
@PopkaAkula yep that problem eluded me for a long time, I cant reproduce it. Do you confirm / infirm @pinesh finding?
What is your monitor refresh rate? Thanks

**Comment by pinesh** [comment]
@tsunamayo at least in my case, this seems to be reproducible. The game boots to my default resolution at either 60 or 120hz (I’m not sure) . I think it’s unlikely to be 240hz as when I select that the game goes very funny (might be more todo with my monitors ability to actually clock 240)x

**Comment by Servomotor** [comment]
I run 2560 x 1440 and take a ton of screenshots at the lowest FOV, but I play at 60 FOV and I have never had the resolution change. 

I run an RTX 2080 if that helps.

**Comment by PopkaAkula** [comment]
> @PopkaAkula yep that problem eluded me for a long time, I cant reproduce it. Do you confirm / infirm @pinesh finding? What is your monitor refresh rate? Thanks

@tsunamayo I can reproduce what @pinesh does and his findings seem to be correct. It does seem to only go to the lowest resolution after you change the resolution setting.
I have a 2560x1440 165Hz monitor.

**Comment by zalexville** [comment]
I also have the issue when switching to 2560x1440 144Hz and leaving the menu in version 22w14a. I am, however, able to set it to 2560x1440 119hz without it defaulting back to 1024p when I leave the menu. I am thinking it might be an issue with pushing beyond 120hz at a higher res?
